### PR TITLE
fix English document of expand/pow/reshape/slice.

### PR DESCRIPTION
**Pull request by liym27**
**1. expand**
image

**2.pow**
image

**3.reshape**
image

**4.slice**
image

### CODE REVIEW

**Review by XiaoguangHu01 — DISMISSED** [review]

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -10918,30 +10918,34 @@ def relu6(x, threshold=6.0, name=None):
...
+
+    :math:`out = x^{factor}`
+
     Args:
-        x(${x_type}): ${x_comment}
-        factor(float|Variable|1.0): The exponential factor of Pow.
-        name(str|None): A name for this layer(optional). If set None, the layer
-                        will be named automatically.
+        x(Variable): An N-D ``Tensor`` or ``LoDTensor`` . The data type is ``float32`` or ``float64``.
+        factor(float32|Variable, optional): A scalar whit type ``float32`` or a ``Tensor`` with shape [1].  The exponential factor of Pow. Default 1.0.
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/nn.py]: A scalar whit type
with

Comment (lanxianghit) [inline python/paddle/fluid/layers/nn.py]: or a ``Tensor`` with shape [1]
需要说明tensor的数据类型

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -11986,43 +11998,55 @@ def slice(input, axes, starts, ends):
...
+        starts (list|tuple|Variable): The data type is ``int32`` . If ``starts`` is a list or tuple, the elements of
+                it should be integers or Tensors with shape [1]. If ``starts`` is an Variable, it should be an 1-D Tensor.
+                It represents starting indices of corresponding axis in ``axes``.
+        ends (list|tuple|Variable): The data type is ``int32`` . If ``ends`` is a list or tuple, the elements of
+                it should be integers or Tensors with shape [1]. If ``ends`` is an Variable, it should be an 1-D Tensor .
+                It represents ending indices of corresponding axis in ``axes``.
 
     Returns:
-        out (Variable): ${out_comment}
+        Variable:  A ``Tensor`` or ``LoDTensor`` with the same dimension as ``input``. The data type is same as ``input``.
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/nn.py]: with the same dimension as ``input``？？

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (liym27) [reply python/paddle/fluid/layers/nn.py]: done

Comment (liym27) [reply python/paddle/fluid/layers/nn.py]: done

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (liym27) [reply python/paddle/fluid/layers/nn.py]: 已修改为 A ``Tensor`` or ``LoDTensor`` with different dimension from ``x``.

**Review by lanxianghit — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]

**Pull request merged**